src/Trigger.h

```diff
@@ -5,7 +5,9 @@
 
 #include "Notifier.h"
 #include "Traverse.h"
+#include "iosource/IOSource.h"
 
+namespace trigger {
```

Comment:
just so i could name the thing `Manager`, and not have it clash with a bunch of other stuff. All of the other managers exist in namespaces (`broker`, `iosource`, etc).

Reply:
Sure seems fine -- still wondering generally if we want to use a top-level `zeek` namespace even for those others, but yeah it's consistent for now and not like the trigger code is expected to be consumed by plugins and thus complicate changing it in the future if we do decide to do that re-namespacing.

Reply:
Quick note: Yes, I think we want to have a `zeek::` namespace eventually, but it may be worth holding off on that until we tackle #266: Switching to a namespace gives a nice opportunity to revisit and define what's public and what not.